blueprints/gke/autopilot/vpc.tf

```diff
@@ -37,6 +37,12 @@ module "vpc" {
   ]
 }
 
+module "firewall" {
```

Comment:
do you need this?

Reply:
yes, to have firewall rule created for IAP